src/registry/map/edit.tsx

```diff
@@ -174,4 +179,38 @@ const UseConfigDefaultMapSettings: React.FC<{}> = () => {
   );
 };
 
+const TileLayer: React.FC = () => {
+  const {getMapTileLayers} = useContext(BuilderContext);
+  const {value: options, loading, error} = useAsync(async () => await getMapTileLayers(), []);
+  if (error) {
+    throw error;
+  }
+
+  const intl = useIntl();
```

Comment:
you must call `useIntl()` before possibly throwing an error, since hooks cannot be called conditionally

```suggestion
  const intl = useIntl();

  if (error) {
    throw error;
  }
```

Reply:
Ah shit, you're right..